src/effects/EffectUI.h

```diff
@@ -14,101 +14,13 @@
 #ifndef __AUDACITY_EFFECTUI_H__
 #define __AUDACITY_EFFECTUI_H__
 
-#include <wx/bitmap.h> // member variables
```

Comment:
I believe it's still required by `EffectUIHost`, and causes compilation issues on Windows platform